apps/vyper/src/app/components/VyperResult.tsx

```diff
@@ -27,18 +26,20 @@ function VyperResult({ output }: VyperResultProps) {
   if (!output) return (
 
     <div id="result">
-      <p>No contract compiled yet.</p>
-      <Button data-id="add-repository" variant="info" onClick={() => remixClient.cloneVyperRepo()}>
-          Clone Vyper repository and play with the contract examples
-      </Button>
+      <p className="my-3">No contract compiled yet.</p>
+      
     </div>
   )
 
   if (isCompilationError(output)) {
     return (
-    <div id="result" className="error">
+    <div id="result" className="error" title={output.message}>
```

Comment:
maybe it would have been bettere to use the react tool tip that we've added.

Reply:
it is not possible to use, it removes the prestyle and needs more space than iframe has.